src/constants/chain-info.ts

```diff
@@ -13,6 +13,8 @@ export interface ChainInfoObject {
   };
 }
 
+export const AvailableBlockChains = ["ETH", "MATIC"];
+
```

Comment:
mmm so for polygon and polygon mumbai we need to use `MATIC` value eh? is this updated at document schema end?